src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4551,6 +4577,12 @@ .ForwardHelpCategory Cmdlet
                 ScopedItemOptions.None),
             // End: Variables which control remoting behavior
 
+            new SessionStateVariableEntry(
+                SpecialVariables.PSSessionApplicationName,
+                "wsman",
+                RemotingErrorIdStrings.PSSessionAppName,
+                ScopedItemOptions.None),
```

Comment:
This is a duplicate and should be removed.

Reply:
fixed